components/sup/src/util/users.rs

```diff
@@ -0,0 +1,142 @@
+// Copyright (c) 2016 Chef Software Inc. and/or applicable contributors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#[cfg(any(target_os="linux", target_os="macos"))]
+extern crate users;
+
+use error::{Result, Error};
+use hcore::package::PackageInstall;
+
+static LOGKEY: &'static str = "USERS";
```

Comment:
By reading the original comment, I presume the LOGKEY in the output is `hab-sup(XX):` ... if so, I too have always wondered and never understood it. As Nathan asked, is this documented? The definitions for the abbreviations ought to be readily available but even so, you're likely still getting that info after the fact (if you can locate it).

What would the extended version look like? e.g. what does `hab-sup(PK):` translate/expand to?